libsigner/src/session.rs

```diff
@@ -214,10 +215,23 @@ impl SignerSession for StackerDBSession {
     /// query the replica for zero or more latest chunks
     fn get_latest_chunks(&mut self, slot_ids: &[u32]) -> Result<Vec<Option<Vec<u8>>>, RPCError> {
         let mut payloads = vec![];
+        let limit = if self.stackerdb_contract_id.name.starts_with("signer") {
+            SIGNERS_STACKERDB_CHUNK_SIZE
```

Comment:
Ah I just noticed this. We should be putting a limit on the miner contract ids as well. I think there is already a global defined max for the miner messages and we should use that when reading miner messages as well.